### PR TITLE
Fixed automatically timer reset in fullscreen

### PR DESCRIPTION
**Pull request by curiale**
The timer state is automatically reset when a new pdf is loaded or it
is reloaded. Otherwise, it should be manually reset by using the reset
button in fullscreen mode

### CODE REVIEW

**Comment by curiale** [comment]
Hi,
Please check the code because I've never programed with cocoa.

**Comment by mpflanzer** [comment]
Sorry, I was quite busy over the last days. I will have a look at it later today!

**Comment by curiale** [comment]
Could you explain me why are you using the initialValue property ? To me, The normal behavior of the reset functionality should be initialize the timer to 0, why are you using the initialValue ?

File: SplitShow/SplitShowController.m [hunk 1]
```diff
@@ -88,6 +88,8 @@ - (BOOL)readFromURL:(NSURL *)file error:(NSError *__autoreleasing *)error
 
         self.presentationMode  = [self getPresentationModeForSlideMode:self.presentation.slideMode];
     }
+  
+    self.timerState = 0;
```
Comment (mpflanzer) [inline SplitShow/SplitShowController.m]: I think there is no need to set it to zero at this point. You added initialisations to all other relevant places.

**Comment by mpflanzer** [comment]
The idea is that one could have a countdown instead of a timer. The class also provides the possibility to set to backwards mode, though it is not implemented for SplitShow at the moment. Simply because I did not need it until now. ;-)

I am wondering if it might be better to have one "global" timer object for all displays instead of creating a new one every time fullscreen mode is entered. Then the timer could also be displayed in non fullscreen mode. Otherwise the timer should have something like a "resetValue" and a "startValue" or "initValue" to keep the countdown possible.

What do you think?

**Comment by curiale** [comment]
Ok, I'm just asking because I couldn't find when initValue sets different to 0. I mean there is no option to initialize the timer yet. Also, I don't think this functionality could be useful.

On the other hand, the idea to have a single timer will be great but I don't know when we could want to have this functionality outside the fullscreen. In my case, I just want to have a memory of the timer due to if I find an error during the presentation.

**Comment by mpflanzer** [comment]
I picked up the idea of having a global timer. It is not contained in the lastest release but already pushed to master.

**Pull request closed without merging**